lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -463,7 +463,7 @@
       - 192.168.1.1
       - 192.168.1.2
     - vlan: 1234
-      type: dhcp
```

Comment:
I think this entry should be dhcp since the second NIC is this example is taking IP configuration from dhcp server.

Reply:
I agree.